TestON/tests/IntentPerfNext/IntentPerfNext.params

```diff
@@ -1,5 +1,5 @@
 <PARAMS>
-    <testcases>1,4,5,2,3,4,5,2,3,4,5,2,3,4</testcases>
+    <testcases>1,4,5,4,5,4,5,4</testcases>
```

Comment:
This seems like the same issue that Shreya encountered last week. I did not change anything in the Perf scripts. Something is wrong with Git.